### PR TITLE
Add com.springrts.SpringLobby

### PR DESCRIPTION
**Pull request by baracoder**
This request is to add SpringLobby, a game lobby for springrts realtime strategy games.
I am a contributor on the github.com/springlobby project (though not a very active one)
https://springlobby.springrts.com/

Reference: https://github.com/springlobby/springlobby/issues/964

### CODE REVIEW

File: com.springrts.SpringLobby.yaml [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+rename-icon: springlobby
+rename-desktop-file: springlobby.desktop
+rename-appdata-file: springlobby.appdata.xml
+finish-args:
+- --share=network
+- --socket=x11
+- --share=ipc
+- --device=dri
+- --socket=pulseaudio
+- --filesystem=home
```
Comment (bilelmoussaoui) [inline com.springrts.SpringLobby.yaml:15]: What does it use home access for?

**Comment by nedrichards** [comment]
bot, build com.springrts.SpringLobby

File: com.springrts.SpringLobby.yaml (hunk 1 above)
Comment (baracoder) [reply com.springrts.SpringLobby.yaml:15]: It has to access `~/.spring/` for game files and `~/.springlobby/` its own settings and game scripts. If it is possible to restrict it, to those directory, this still should work.

Comment (barthalion) [reply com.springrts.SpringLobby.yaml:15]: You can use `--persist=.spring` and `--persist=.springlobby` instead.

**Comment by barthalion** [comment]
Appdata file needs some changes:

```
builddir/files/share/appdata/com.springrts.SpringLobby.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• attribute-invalid     : <screenshot> width (892) did not match specified (1920) [https://springlobby.springrts.com/screenshots/01.png]
• attribute-invalid     : <screenshot> height (751) did not match specified (1056) [https://springlobby.springrts.com/screenshots/01.png]
• attribute-invalid     : <screenshot> width (892) did not match specified (1920) [https://springlobby.springrts.com/screenshots/02.png]
• attribute-invalid     : <screenshot> height (751) did not match specified (1056) [https://springlobby.springrts.com/screenshots/02.png]
• tag-missing           : <release> required
```

**Comment by baracoder** [comment]
Thanks for checking it. I have replaced `--fileystem=home` with `--persist` and added a new version of the appdata.xml file

**Comment by barthalion** [comment]
bot, build com.springrts.SpringLobby

File: com.springrts.SpringLobby.yaml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+- name: springlobby
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://github.com/springlobby/springlobby.git
+    tag: '0.270'
+    commit: '48257e6890e360e5d947a3fee68d0d48b2ca1846'
+  - type: file
+      path: springlobby.appdata.xml
```
Comment (barthalion) [inline com.springrts.SpringLobby.yaml:96]: This needs to be on `type` indentation level.

File: com.springrts.SpringLobby.yaml (hunk 1 above)
Comment (baracoder) [reply com.springrts.SpringLobby.yaml]: Thanks, changed `--filesystem=home` to `--persist=.spring` and `--persist=.springlobby`

File: com.springrts.SpringLobby.yaml (hunk 2 above)
Comment (baracoder) [reply com.springrts.SpringLobby.yaml]: Sorry, forgot to fixup that commit

**Comment by baracoder** [comment]
bot, build com.springrts.SpringLobby

**Comment by baracoder** [comment]
bot, build com.springrts.SpringLobby

**Comment by barthalion** [comment]
/merge

Let us know if we should add someone more from the team to the Flathub repo. Thanks!

**Comment by baracoder** [comment]
@barthalion thank you. Please add following members: @abma and @renefritze

**Comment by barthalion** [comment]
Invitations sent.

**Pull request closed without merging**